packages/ListBox/src/store/Provider.js

```diff
@@ -66,12 +66,14 @@ function initializeState(props) {
     formElementId,
     formElementLabelDescribedBy,
     size,
+    hasFilter: false,
+    lastKnownSelectedOptions: selectedOptions.slice(0),
+    onChangeFn: null,
```

Comment:
Can you please add these alphabetically? The others are (with the exception of `formElementId` and `formElementLabelDescribedBy`... maybe fix up those two too please)

Reply:
👍 done!